.eslintignore

```diff
@@ -10,3 +10,4 @@ src/hca-migrate/*
 src/rx-migrate/*
 coverage/*
 .nyc_output/*
+src/js/common/utils/static-page-widgets.js
```

Comment:
This file isn't transpiled and comments are shipped to users, so I opted to ignore it here.